### DISCUSSION TITLE
Which config file should be selected?

### DISCUSSION THREAD

**Discussion by aubreymoore** [plazi/community#83] [Q&A] [answered]
The **commented list of steps** suggests selecting **Default.imagine | Local** in Viewing mode and 
**Default.imagine | GgServer** in Suggestion mode. Which is correct?

In Viewing mode:

1.b. Always select the option “Default.imagine” and click “OK”. If more than one “Default.imagine” option is available, click the one with the ‘Local’ value on the “Config. Host” column.

In Suggestion mode:

![image](https://user-images.githubusercontent.com/1120591/73686968-c3e98300-4714-11ea-99b0-a41ceab5fb0b.png)

**Comment by flsimoes** [comment, marked as answer]
Hi Aubrey,

Suggestion mode is the correct one (selected third option in the image below), sorry for the confusion.
![image](https://user-images.githubusercontent.com/3343086/73748072-7a5e5f80-4737-11ea-81b4-84aaab6d2b85.png)

We'll write this one more clearly in the next version thanks to your input!

**Comment by mguidoti** [comment]
Hi @aubreymoore,

I'm the one slacking this document. I've to review it, just didn't had the time yet. But @flsimoes correctly replied your question.

Thanks,

**Comment by mguidoti** [comment]
Hi @aubreymoore,

I'm the one slagging this document. I've to review it, just didn't had the time yet. But @flsimoes correctly replied your question.

Thanks,
